### PR TITLE
Add remaining stub tests

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: test/functions/projectFunctionsTest.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+          'usdBalance': 2319334.639906516,
+          'volumeUsd': 14141933.4691686
+        }
+      ]
+    )
+    const response = san.allProjects_()
+
+    expect(response).to.deep.eq(
+      [
+        [ 'Ticker', 'Name', 'Slug', 'USD Price', 'USD Marketcap', 'USD Volume', 'USD Balance', 'ETH Balance', 'ETH Spent 30D', 'ETH Spent 7D', 'ETH Spent 1D' ],
```
Comment (kalkov) [inline test/functions/projectFunctionsTest.js]: Too long to read

File: test/functions/projectFunctionsTest.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+          'usdBalance': 2327365.1751939454,
+          'volumeUsd': 14263628.1570071
+        }
+      ]
+    )
+    const response = san.erc20Projects_()
+
+    expect(response).to.deep.eq(
+      [
+        [ 'Ticker', 'Name', 'Slug', 'USD Price', 'USD Marketcap', 'USD Volume', 'USD Balance', 'ETH Balance', 'ETH Spent 30D', 'ETH Spent 7D', 'ETH Spent 1D', 'Main Contract Address' ],
```
Comment (kalkov) [inline test/functions/projectFunctionsTest.js]: Too long to read

File: test/functions/projectFunctionsTest.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+          'USD Price',
+          'USD Volume',
+          'USD Marketcap',
+          'Percent Change 24H',
+          'Percent Change 7D',
+          'Volume Change 24H',
+          'Available Supply',
+          'Average Dev Activity 30D'
+        ],
+        [ 'SAN', 'Santiment', 'santiment', 9155789.4, 0, 29583.389448238777, 5635387.2655066, 0.47166911299, 15310.6778037951, 29554961.588902876, 0.319887, -10.7174, 21.95, 62660370.9569795, 71.5 ]
```
Comment (kalkov) [inline test/functions/projectFunctionsTest.js]: Too long to read

File: test/functions/projectFunctionsTest.js [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+          'Whitepaper Link',
+          'Facebook Link',
+          'Blog Link',
+          'LinkedIn Link',
+          'Github Link',
+          'Twitter Link',
+          'Reddit Link',
+          'Chat Link'
+        ],
+        [ 'SAN', 'Santiment', 'santiment', 'https://santiment.net/', 'https://docs.google.com/document/d/1hHmJQWrPrObSgCA8kTbk-4LgqGbYzukQAjtg5b8jwc0/view', 'https://www.facebook.com/santimentfeed/', 'https://santiment.net/blog/', null, 'https://github.com/santiment', 'https://twitter.com/santimentfeed', 'https://www.reddit.com/r/santiment/', 'https://discord.gg/6fMxk82' ]
```
Comment (kalkov) [inline test/functions/projectFunctionsTest.js]: Too long

Comment (spiderjako) [reply test/functions/projectFunctionsTest.js]: They should be okay now

**Pull request merged**